### DISCUSSION TITLE
[Feature Request]: Find & Replace Feature for Editor

### DISCUSSION THREAD

**Discussion by RomikMakavana** [ueberdosis/tiptap#4563] [Feature Requests]
### Description

Hello Developers ,
 
There is Find and Replace Feature in V1 , but after that it was removed and still not added.

and in this era , find & Replace is one of the key feature of the editor.

I would like to propose the addition of a 'Find & Replace' feature to the Tiptap editor. This feature would allow users to search for specific text within the editor's content and replace it with new text. 'Find & Replace' is a common and essential functionality in text editors, and its addition to Tiptap would enhance the editor's usability.

### Use Case

1. Users can search for specific words or phrases within the editor's content.
2. Users can replace individual instances or all occurrences of the found text with the desired replacement.

### Type

New extension

**Comment by ImageDeeply** [comment]
FYI: https://github.com/ueberdosis/tiptap/pull/2075

(No affiliation; just something I came across while searching.)

**Reply by nperez0111** [reply]
Yep would encourage this: https://github.com/sereneinserenade/tiptap-search-and-replace
